working-groups/comms/charter.md

```diff
@@ -0,0 +1,61 @@
+# TAG ENV Working Group Communications Charter
+## Motivation
+Marketing is as crucial as code to any open source project’s success. Participation in open source projects play a vital role in developing a sustainable ecosystem around a project by marketing the project through their networks. The key is to promote the project first and always remain authentic and true to the open source ethos of openness and transparency.
+
+This is a proposal to create a Working Group under the CNCF Environmental Sustainability Technical Advisory Group (TAG ENV) to help organize communication and marketing efforts.
```

Comment:
Since this is official definition of the working group we should remove the part about this being a proposal.

Reply:
proposed changes: https://github.com/cncf/tag-env-sustainability/pull/247#discussion_r1364530346